packages/examples/simple/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "demo": "node -r @parcel/babel-register ./node_modules/.bin/parcel src/index.js"
+    "demo": "node -r @parcel/babel-register ./node_modules/.bin/parcel src/index.js -w"
```

Comment:
watching isn't working apparently

Reply:
heh, this also just moved to the new cli interface in #2860 

I think it would be `parcel2 watch src/index.js` there?

Reply:
Yeah saw that, will update once that PR is merged